Rakefile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Is this from rubocop? Looks like a good thing, we should add it to the skel if we're going to do it to all repos.